pkg/kubelet/apis/config/fuzzer/fuzzer.go

```diff
@@ -110,6 +110,8 @@ func Funcs(codecs runtimeserializer.CodecFactory) []interface{} {
 			}
 			obj.EnableSystemLogHandler = true
 			obj.MemoryThrottlingFactor = utilpointer.Float64Ptr(rand.Float64())
+			obj.ExitOnLockContention = true
```

Comment:
I think we only needed to provide fixed values for fields where the defaulter will change it from the zero-value to some other value (which messes up the fuzzed round trip). Since the defaults for these fields are the zero-value, I don't think we have to set them here.

Reply:
@mtaufen Hey sorry, this work got delayed due to the pre-requisite e2e test.

I'm not completely sure about this but reading the comment on this [line](https://github.com/ipochi/kubernetes/blob/b5be8e42467966bdf8f2c33674f430e0d857cc28/pkg/kubelet/apis/config/fuzzer/fuzzer.go#L39) makes your point valid.